models/Class.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Enum for the weekday field of class
+ */
+export enum weekdays {
+    MON = "MON",
+    TUE = "TUE",
+    WED = "WED",
+    THU = "THU",
+    FRI = "FRI",
+    SAT = "SAT",
+    SUN = "SUN",
+}
+
+/**
+ * Request Body Input for POST /class
+ */
+export type createClassInput = {
+    name: string;
```

Comment:
Add `?` to the fields that are optional

Reply:
@mahadzaryab1 Is this required actually?

Reply:
yeah - the fields that are optional should have a `?`. It then sets them to undefined if a value isn't provided.